apps/website/src/server/db/users.ts

```diff
@@ -1,20 +1,10 @@
 import { prisma } from "@/server/db/client";
 
-const cache = new Map<string, string[]>();
-
 export async function getRolesForUser(userId: string) {
-  const cachedRoles = cache.get(userId);
-  if (cachedRoles) {
-    return cachedRoles;
-  }
-
-  const roles = (
+  return (
     await prisma.userRole.findMany({
+      select: { role: true },
```

Comment:
What does this do? Just returning the role column only?

Reply:
yeah. does not really make any difference, but its `select role` instead of `select *` since we only need the role